vendor/profile.ps1

```diff
@@ -5,13 +5,33 @@ if( -not $env:PSModulePath.Contains($CmderModulePath) ){
     $env:PSModulePath = $env:PSModulePath.Insert(0, "$CmderModulePath;")
 }
 
+try {
+    Get-command -Name "git" -ErrorAction Stop >$null
+    Import-Module -Name "posh-git" -ErrorAction Stop >$null
+    $gitStatus = $true
+} catch {
+    Write-Warning "Missing git support"
+    $gitStatus = $false
+}
+
+function checkGit($Path) {
```

Comment:
I pulled out the code that imported everything all the time from our `vendor\psmodules` in another commit (e811c60ab2433405c0cf12843f56eb2e16425b41)

Posh-git needs a module manifest creating to behave with powershell's auto-loading behaviour. This checks if we have git and need the cmdlet only if we're actually in a git repository. 

All in the name of speed. You don't need to wait for all the git stuff constantly especially if its not even used.